src/main/java/org/gridsuite/study/server/service/NetworkModificationTreeService.java

```diff
@@ -932,4 +932,37 @@ public long countBuiltNodes(UUID studyUuid, UUID rootNetworkUuid) {
         // perform N queries, but it's fast: 25 ms for 400 nodes
         return nodes.stream().filter(n -> self.getNodeBuildStatus(n.getIdNode(), rootNetworkUuid).isBuilt()).count();
     }
+
+    @Transactional
+    public List<NodeAlias> getNodeAliases(UUID nodeId) {
+        NetworkModificationNodeInfoEntity node = getNetworkModificationNodeInfoEntity(nodeId);
```

Comment:
this throws an exception if nodeId is the root node ...

Reply:
I was wrongly linking the table node_alias to network_modification_node_info, changed the link to study table

Reply:
After discussing it the model has been simplified by allowing only one alias per node, the data is now directly stored in node table